### PR TITLE
RTEMIS-154: Fixed the People Page.

### PR DESCRIPTION
**Pull request by RohanSingh-inno**
[1] Modified the breadcrumbs.
[2] Modified the role field for people pagel.

Ticket # : https://innoraft.atlassian.net/browse/RTEMIS-154

### CODE REVIEW

File: config/install/easy_breadcrumb.settings.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ applies_admin_routes: true
...
...n/admin/school/mapping :: Schools | /admin/school :: School-Mapping\r\n/admin/student :: Students\r\n/admin/content/mini_node/add/academic_session :: Academic Sessions List|/academic-sessions :: Academic Session\r\n/admin/structure/taxonomy/manage/school/add :: Add School\r\n/admin/structure/taxonomy/manage/school/overview :: School List\r\n/admin/config/regional/translate :: User Interface Translation\r\n/admin/people :: People\r\n/admin/people/create :: People | /admin/people :: Add User\r\n/admin/location/bulk-upload :: Location :: Bulk Upload\r\n/admin/structure/taxonomy/manage/location/overview :: Location List\r\n/user/register :: School Register\r\nregex!/school/registration/([^/]*)/([^/]*)/edit :: Edit School Registration\r\nregex!/school/registration/([^/]*)/([^/]*)/view :: School Registration\r\n/student/application :: Student-Application\r\n/admin/content/mini_node/add/student_details ::  Add Student Details\r\n/student/login :: Student-Login\r\nregex!/user/([^/]*)/edit :: Edit\r\n/student/application-status :: Student Application Status\r\n/admin/structure/taxonomy/manage/location/add :: Location List | /admin/structure/taxonomy/manage/location/overview :: Add Location"
```
Comment (kunal-sin) [inline config/install/easy_breadcrumb.settings.yml:5]: Didn't get this
admin/structure/taxonomy/manage/location/add :: Location List 
admin/structure/taxonomy/manage/location/overview :: Add Location

Don't you think it should be opposite?

Comment (RohanSingh-inno) [reply config/install/easy_breadcrumb.settings.yml:5]: We are showing the Location list tot he state admin, from that we have the option to add location.
So the breadcrumb should be like HOME >> Location List >> Add Location

Comment (kunal-sin) [reply config/install/easy_breadcrumb.settings.yml:5]: `admin/structure/taxonomy/manage/location/add` This url lands on the location add page, right?

Comment (RohanSingh-inno) [reply config/install/easy_breadcrumb.settings.yml:5]: yes,
/admin/structure/taxonomy/manage/location/add :: Location List | /admin/structure/taxonomy/manage/location/overview :: Add Location". Please check the syntax here, | denotes the Breadcrumb Link Name(separated by the destination link).

Comment (kunal-sin) [reply config/install/easy_breadcrumb.settings.yml:5]: Share the SS

Comment (RohanSingh-inno) [reply config/install/easy_breadcrumb.settings.yml:5]: image
image

Comment (kunal-sin) [reply config/install/easy_breadcrumb.settings.yml:5]: Cool, Thanks. Understood

**Pull request merged**